src/CoreBundle/private/library/classes/TCMSTableEditor/TCMSTableEditorEndPoint.class.php

```diff
@@ -2093,41 +2093,53 @@ public function DeleteIdConnectedRecordReferences()
     protected function deleteMultiTableRecordReferences(string $tableName, string $id = ''): void
     {
         $databaseConnection = $this->getDatabaseConnection();
+        $editLanguage = $this->getLanguageService()->getActiveEditLanguage();
```

Comment:
Both initializations are not necessary before the early return.

Reply:
Right. moved.